app/v1/views/redflag_view.py

```diff
@@ -0,0 +1,84 @@
+import uuid, datetime, requests
+from flask import request,jsonify, make_response, Blueprint, abort, json
+from rapidjson import loads
+from flask_restful import Resource
+from app.v1.models.red_flag_model import RedFlagModel
+
+red_flag_api = Blueprint('red_flag_api',__name__)
+
+
+@red_flag_api.route('/',methods=['GET'])
+@red_flag_api.route('/redflag/',methods=['GET'])
+def get_all():
+    """"Get all red_flags"""
+    all_flags = RedFlagModel().get_red_flags()
+    response_obj = {
+        'status': 200,
+        'data': [all_flags]
+    }
+    return make_response(jsonify(response_obj),200)
+
+
+@red_flag_api.route('/',methods=['POST','GET'])
+@red_flag_api.route('/redflag/',methods=['GET', 'POST'])
+def redflags():
+    if request.method == 'POST':
+        new_red_flag = RedFlagModel()
+        json_data = request.get_json(force=True)
+        payload = dict(
+            created_by = json_data['created_by'],
+            record_type = json_data['record_type'],
+            location = json_data['location'],
+            status = json_data['status'],
+            image = json_data['image'],
+            video = json_data['video'],
+            comment = json_data['comment'],
+        )
+        new_red_flag.save(payload)
+        response_obj = {
+                'status': 201,
+                'message': "Created red-flag record"
+        }
+        return make_response(jsonify(response_obj),201)
+
+    else:
+        # GET
+        all_flags = new_red_flag.get_all()
+        response_obj = {
```

Comment:
I don't think it is neccessary to have this response_obj

Reply:
I think it is better to store the response payload in a variable, then it would be easy to parse into jsonify